frontend/src/metabase/components/Calendar.unit.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import moment from "moment-timezone";
+import { render, screen } from "__support__/ui";
+import MetabaseSettings from "metabase/lib/settings";
+import { updateMomentStartOfWeek } from "metabase/lib/i18n";
+import Calendar from "./Calendar";
+
+describe("Calendar", () => {
+  it("should render weekday short names", () => {
+    setup();
+
+    expect(
+      screen
+        .getAllByTestId("calendar-day-name")
+        .map((dayEl, index) => dayEl.textContent),
+    ).toEqual(["Su", "Mo", "Tu", "We", "Th", "Fr", "Sa"]);
+  });
+
+  it("should render days based on first day of the week settings", () => {
+    MetabaseSettings.set("start-of-week", "wednesday");
```

Comment:
Could you move that to `afterEach` to make sure that it gets set even if the test fails?